### PR TITLE
Localize Terms of Use and Privacy Policy pages

### LINKED ISSUE #461: localize privacy policy
In #459 , the translations for the about page markdown were added to the common package. We're now ready to enable localization for our privacy policy.

## Translations

https://docs.google.com/document/d/1DdlQjrvZSGV82M5u7ab3pPlKOlmzl-6Tf4msMUSB0Ow/edit?usp=sharing

## Content

https://github.com/CodeForBoulder/upswyng/blob/master/packages/common/src/content/PrivacyPolicy.md

## Checklist
- [ ] translations for both English and Spanish were added to the shared package
- [ ] translation keys were substituted for the correct web package content
- [ ] the server package was updated to import and use the english version of this content

### PR DESCRIPTION
**Pull request by cjlynch278**
This PR closes #461 and #462

## What does this PR do?
Adds the translations for the Terms of Use and Privacy Policy pages

## How does this PR make you feel? [:link:](http://giphy.com/categories/emotions/)

I'm still new to this project so please carefully inspect my changes - I don't have my environment properly set up to test.

Additionally I added the translation keys/mappings for the pages in the packages/common/src/i18n/en.ts/es.ts files under the "About" category. I know this may be a tad misleading, but I thought I'd group the 3 pages together.

### CODE REVIEW

File: packages/common/src/content/terms-of-use/es.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+### Restricciones.
+
+No puedes: (i) eliminar ningún derecho de autor, marca comercial u otros avisos de propiedad en ninguna parte de los Servicios; (ii) reproducir, modificar, preparar projectos relacionados basados ​​en, distribuir, licenciar, arrendar, vender, revender, transferir, exhibir públicamente, ejecutar públicamente, transmitir, transmitir, difundir o explotar de otra manera los Servicios, excepto según lo expresamente explicitamente por UpSwyng; (iii) descompilar, realizar ingeniería inversa o desensamblar los Servicios, excepto según lo permitido por la ley; (iv) vincular, reflejar o enmarcar cualquier parte de los Servicios; (v) causar o lanzar programas o scripts con el propósito de raspar, indexar, inspeccionar o extraer datos de cualquier parte de los Servicios o sobrecargar u obstaculizar indebidamente el funcionamiento y/o la funcionalidad de cualquier aspecto de los Servicios; o (vi) intentar obtener acceso no autorizado o dañar cualquier aspecto de los Servicios o sus sistemas o redes relacionadas.
+
+#### Propiedad.
```
Comment (jacobvenable) [inline packages/common/src/content/terms-of-use/es.md:15]:
```suggestion
### Propiedad.
```
Nit, just a small heading level discrepancy.

**Review by jacobvenable — COMMENTED** [review]
Usually we aim for having a single PR per issue to separate concerns, but that's not a big deal.

There are a couple of small nits to address before merge, but I think this looks really good! 😄 

I saw your note about having issues setting up the environment, so let me know if you would like help troubleshooting.

File: packages/common/src/i18n/en.ts [hunk 2]
```diff
@@ -1,8 +1,12 @@
 import { AboutContent } from "@upswyng/common";
+import { PrivacyPolicyContent } from "@upswyng/common";
+import { TermsOfUseContent } from "@upswyng/common";
 
 export default {
   about: {
     content: AboutContent.en,
+    privacyPolicy: PrivacyPolicyContent.en,
+    termsOfUse: TermsOfUseContent.en,
```
[the remark this answers is not in the document]
Comment (cjlynch278) [reply packages/common/src/i18n/en.ts]: I added these in alphabetical order. I also added the changes to the PrivacyPolicy.tsx and TermsOfUse.tsx files

**Comment by cjlynch278** [comment]
Hey @jacobvenable I'm pretty sure I integrated all of the changes that you had mentioned and also a changed the files that are calling the changed keys. 

I will definitely take you up on your offer to set up my dev environment. I'll try to ping you if we both happen to be online.

**Review by jacobvenable — APPROVED** [review]
These changes looks great! Thanks @cjlynch278 !! 🌮 🚀

**Pull request merged**